README.asciidoc

```diff
@@ -201,3 +201,62 @@ Run individual tests by specifying them explicitly:
 ----
 make check TESTS=23-baseclass.t
 ----
+
+How to run test cases
+---------------------
+
+This following instructions shows how to run test cases using the openSUSE test
+distribution as an example.
+
+Clone the test distribution and needles:
+
+-----------------------------------------------------------------------------
+mkdir distri
+cd distri
+git clone git@github.com:os-autoinst/os-autoinst-distri-opensuse.git opensuse
+cd opensuse
+git clone git@github.com:os-autoinst/os-autoinst-needles-opensuse.git needles
```

Comment:
That will actually not work as needles would need to be cloned into the products subdir but please follow my above suggestion